routes/index.js

```diff
@@ -17,5 +17,6 @@ app.use("/trade", require("./trading.js"));
 app.use("/users", require("./users.js"));
 app.use("/profileDiffs", require("./profileDiffs.js"));
 app.use("/wallet", require("./wallets.js"));
+app.use("/signout", require("./signout.js"));
```

Comment:
Shouldn't the route for this be : `/auth/signout` instead of `/signout` ?

Reply:
this is a good point, let me check this